config/checkstyle_input_suppressions.xml

```diff
@@ -161,27 +161,9 @@
   <suppress id="ConfigCommentOnTopInputs"
             files="coding[\\/]multiplestringliterals[\\/]InputMultipleStringLiteralsNoWarnings.java"/>
 
-  <suppress id="ConfigCommentOnTopInputs"
-            files="coding[\\/]packagedeclaration[\\/]InputPackageDeclarationDiffDirectoryAtParent.java"/>
-
-  <suppress id="ConfigCommentOnTopInputs"
-            files="coding[\\/]packagedeclaration[\\/]InputPackageDeclarationDiffDirectoryAtSubpackage.java"/>
-
-  <suppress id="ConfigCommentOnTopInputs"
-            files="coding[\\/]packagedeclaration[\\/]InputPackageDeclarationDiffDirectory.java"/>
-
   <suppress id="ConfigCommentOnTopInputs"
             files="coding[\\/]packagedeclaration[\\/]InputPackageDeclarationEmptyFile.java"/>
```

Comment:
Can't do anything for this one. File is meant to be empty.

Reply:
keep it , we will place comment on it to explain that it is not forgoten